Controllers/address.js

```diff
@@ -0,0 +1,50 @@
+const _ = require('lodash');
+exports.getAddress = async (req, res) => {
+	const { address } = req;
+	try {
+		res.json({ response: address });
+	} catch {
+		res.status(401).json({ response: error.message });
+	}
+};
+
+exports.addAddress = async (req, res) => {
+	const { address } = req;
+	const addAddress = req.body;
+	try {
+		const updatedAddress = _.extend(address, {
+			addresses: _.concat(address.addresses, { ...addAddress }),
+		});
+		await updatedAddress.save();
+		res.json({ response: updatedAddress });
+	} catch (error) {
+		res.status(401).json({ response: error.message });
+	}
+};
+
+exports.updateAddress = async (req, res) => {
+	const { address } = req;
+	const { addressId } = req.params;
+	const addressFromBody = req.body;
+	let updateAddress = address.addresses.id(addressId);
+	updateAddress = _.extend(updateAddress, { ...addressFromBody });
```

Comment:
I think `{ ...addressFromBody }` destructuring is not required here

Reply:
done